interbotix_ros_xscobots/interbotix_xscobot_moveit/launch/moveit_api.launch.py

```diff
@@ -0,0 +1,355 @@
+    except (
+        EnvironmentError
+    ):  # parent of IOError, OSError *and* WindowsError where available
+        return None
+
+
+def launch_setup(context, *args, **kwargs):
+    robot_model_launch_arg = LaunchConfiguration('robot_model')
+    robot_name_launch_arg = LaunchConfiguration('robot_name')
+    base_link_frame_launch_arg = LaunchConfiguration('base_link_frame')
+    use_world_frame_launch_arg = LaunchConfiguration('use_world_frame')
+    show_ar_tag_launch_arg = LaunchConfiguration('show_ar_tag')
+    external_urdf_loc_launch_arg = LaunchConfiguration('external_urdf_loc')
+    mode_configs_launch_arg = LaunchConfiguration('mode_configs')
+    rviz_frame_launch_arg = LaunchConfiguration('rviz_frame')
+    robot_description_launch_arg = LaunchConfiguration('robot_description')
+    hardware_type_launch_arg = LaunchConfiguration('hardware_type')
+    xs_driver_logging_level_launch_arg = LaunchConfiguration('xs_driver_logging_level')
+
+    use_sim_time_param = determine_use_sim_time_param(
+        context=context, hardware_type_launch_arg=hardware_type_launch_arg
+    )
+
+    remappings = [
+        (
+            f'{robot_name_launch_arg.perform(context)}/get_planning_scene',
+            f'/{robot_name_launch_arg.perform(context)}/get_planning_scene',
+        ),
+        (
+            '/arm_controller/follow_joint_trajectory',
+            f'/{robot_name_launch_arg.perform(context)}'
+            '/arm_controller/follow_joint_trajectory',
+        ),
+        (
+            '/gripper_controller/follow_joint_trajectory',
+            f'/{robot_name_launch_arg.perform(context)}'
+            '/gripper_controller/follow_joint_trajectory',
+        ),
+    ]
+
+    planning_scene_monitor_parameters = {
+        'publish_planning_scene': True,
+        'publish_geometry_updates': True,
+        'publish_state_updates': True,
+        'publish_transforms_updates': True,
+    }
+
+    moveit_config = (
+        MoveItConfigsBuilder(
+            robot_name='dx400', package_name='interbotix_xscobot_moveit'
+        )
+        .robot_description(file_path='config/dx400.urdf.xacro')
+        .robot_description_semantic(file_path='config/srdf/dx400.srdf.xacro')
+        .trajectory_execution(file_path='config/moveit_controllers.yaml')
+        .joint_limits(file_path='config/joint_limits/dx400_joint_limits.yaml')
+        .robot_description_kinematics(file_path='config/moveit_kinematics.yaml')
+        .planning_scene_monitor(planning_scene_monitor_parameters)
+        # .planning_pipelines(pipelines=['ompl'])
+        .moveit_cpp(
+            file_path=get_package_share_directory('interbotix_moveit_interface')
+            + '/config/planning.yaml'
+        )
+        .pilz_cartesian_limits(file_path='config/pilz_cartesian_limits.yaml')
+        .to_moveit_configs()
+    )
+
+    example_file = DeclareLaunchArgument(
+        'example_file',
+        default_value='example.py',
+        description='Python API tutorial file name',
```

Comment:
Either hardcode the filepath into this or make the description here more generic. I'm fine with hardcoding this since it's a demo.

Reply:
Fixed it.